packages/dd-trace/src/ci-visibility/dynamic-instrumentation/worker/index.js

```diff
@@ -69,14 +71,20 @@ async function addBreakpoint (snapshotId, probe) {
   const script = findScriptFromPartialPath(file)
   if (!script) throw new Error(`No loaded script found for ${file}`)
 
-  const [path, scriptId] = script
+  const [path, scriptId, sourceMapURL] = script
 
   log.debug(`Adding breakpoint at ${path}:${line}`)
 
+  let generatedPosition = { line }
+
+  if (sourceMapURL && sourceMapURL.startsWith('data:')) {
+    generatedPosition = await processScriptWithInlineSourceMap({ file, line, sourceMapURL })
+  }
```

Comment:
this handles inline sourcemaps so that we can add a line probe where the user wants, instead of the compiled code